setup.py

```diff
@@ -164,19 +167,17 @@ def get_extras_require() -> Dict[str, List[str]]:
             "keras",
             "tensorflow",
             "tensorflow-datasets",
+            "pytorch-ignite",
+            "pytorch-lightning>=0.8.1",
+            "skorch",
+            "catalyst",
         ]
         + (
-            (
-                ["torch==1.6.0", "torchvision==0.7.0"]
-                if sys.platform == "darwin"
-                else ["torch==1.6.0+cpu", "torchvision==0.7.0+cpu"]
-            )
-            + ["pytorch-ignite", "pytorch-lightning>=0.8.1", "skorch"]
-            if (3, 5) < sys.version_info[:2]
-            else []
+            ["torch==1.6.0", "torchvision==0.7.0"]
+            if sys.platform == "darwin"
+            else ["torch==1.6.0+cpu", "torchvision==0.7.0+cpu"]
         )
-        + (["allennlp==1.0.0", "fastai<2"] if (3, 5) < sys.version_info[:2] < (3, 8) else [])
-        + (["catalyst"] if (3, 5) < sys.version_info[:2] else []),
+        + (["allennlp==1.0.0", "fastai<2"] if sys.version_info[:2] < (3, 8) else []),
     }
 
     return requirements
```

Comment:
Inserting an assertion like this can prevent duplicating packages.

```python
def contains_duplicates(arr):
    return len(arr) == len(set(arr))

for packages in requirements.values():
    assert contains_duplicates(map(get_package_name, packages))

return requirements
```

Reply:
That's one way to ensure it. Not a strong opinion, but I think we can still manage these duplicates in review and that it sounds a bit like over-engineering. Please correct me.